build.sbt

```diff
@@ -43,7 +43,8 @@ lazy val kamuCli = (project in file("."))
       "sqlline" % "sqlline" % "1.8.0",
       // NOTE: Using kamu-specific Hive version with some bugfixes
       // remove `kamu.X` part if you want a simpler build
-      ("org.spark-project.hive" % "hive-jdbc" % "1.2.1.spark2.kamu.1")
+      // ("org.spark-project.hive" % "hive-jdbc" % "1.2.1.spark2.kamu.1")
+      ("org.spark-project.hive" % "hive-jdbc" % "1.2.1.spark2")
```

Comment:
Is this intentional? Wouldn't want to accidentally make a release with a wrong dependency... We need to research how to specify a version dynamically.

Reply:
Please clarify. Dynamically as in 'it is not bound to a particular version', or 'there is a flag we pass to the build tool to choose between pre-specified dependencies'?.

Reply:
Either way I don't know how to do that just yet, so I just created the issue:
https://github.com/kamu-data/kamu-cli/issues/10

Reply:
Let's revert this and use `-Dhive.jdbc.version=upstream` in travis file instead.

Reply:
Done